utilities/PerformanceTesting/query.py

```diff
@@ -0,0 +1,293 @@
+
+    return res
+
+
+def get_cpu_mem_network_usage(querier, query_window):
+    """
+    Performs Prometheus queries relevant to cpu, memory, and network IO usage
+
+    :param querier: An object containing information
+    needed to query the Prometheus API
+    :param query_window: The window over which rates are computed by Prometheus
+
+    :return: Json containing information about cpu, memory,
+    and network IO usage
+    """
+    res = json.loads('{}')
+    metric_names = {
+            'container_network_receive_bytes_total':
+            'network received (bytes per second)',
+            'container_network_transmit_bytes_total':
+            'network transmited (bytes per second)'
+            }
+
+    for stat in stat_names:
+        cpu_query = stat + ' by (job) (rate(container_cpu_usage_seconds_total'
+        cpu_query += '{namespace="stackrox"}[' + query_window + ']) * 100)'
+
+        cpu_query_name = stat_names[stat] + ' cpu usage'
+
+        mem_query = stat + ' by (job) '
+        mem_query += '(container_memory_usage_bytes{namespace="stackrox"})'
+
+        mem_query_name = stat_names[stat] + ' mem usage'
+
+        res[cpu_query_name] = querier.query_and_get_value(cpu_query)
+        res[mem_query_name] = querier.query_and_get_value(mem_query)
+
+    for name in metric_names:
+        for stat in stat_names:
+            query = stat + ' by (job) (' + name + '{namespace="stackrox"})'
+            query_name = stat_names[stat] + ' total '
+            query_name += metric_names[name] + ' (bytes)'
+
+            rate_query = stat + ' by (job) (rate(' + name
+            rate_query += '{namespace="stackrox"}[' + query_window + ']))'
+            rate_query_name = stat_names[stat] + ' '
+            rate_query_name += metric_names[name] + ' (bytes per second)'
+
+            res[query_name] = querier.query_and_get_value(query)
+            res[rate_query_name] = querier.query_and_get_value(rate_query)
+
+    return res
+
+
+def main(query_window, token, url, output_file):
+    """
+    Performs Prometheus queries related to the performance of Collector
+    and StackRox
+
+    :param query_window: The window over which rates are computed by Prometheus
+    :param token: The token used to authenticate for the Prometheus API
+    :param url: The Prometheus API endpoint
+    :output_file: Where the results are written to
+    """
+    querier = Querier(token, url)
+
+    collector_timers = get_collector_timers(querier)
+    collector_counters = get_collector_counters(querier)
+    sensor_network_flows = get_sensor_network_flows(querier, query_window)
+    cpu_mem_network_usage = get_cpu_mem_network_usage(querier, query_window)
+
+    json_output = {
+                    'collector_timers': collector_timers,
+                    'collector_counters': collector_counters,
+                    'sensor_network_flows': sensor_network_flows,
+                    'cpu_mem_network_usage': cpu_mem_network_usage
+    }
```

Comment:
[nit] can we PEP8 this file

Reply:
This is pep8 compliant 

pep8 --first query.py
/usr/local/lib/python3.6/site-packages/pep8.py:2124: UserWarning: 

pep8 has been renamed to pycodestyle (GitHub issue #466)
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.

$ pip install pycodestyle
$ pycodestyle ...

  '\n\n'



pep8 --first GetAverages.py 
/usr/local/lib/python3.6/site-packages/pep8.py:2124: UserWarning: 

pep8 has been renamed to pycodestyle (GitHub issue #466)
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.

$ pip install pycodestyle
$ pycodestyle ...

  '\n\n'